husarion_ugv_manager/include/husarion_ugv_manager/lights_manager_node.hpp

```diff
@@ -71,6 +72,7 @@ class LightsManagerNode : public rclcpp::Node
 private:
   void BatteryCB(const BatteryStateMsg::SharedPtr battery);
   void EStopCB(const BoolMsg::SharedPtr e_stop);
+  void JoyCB(const sensor_msgs::msg::Joy::SharedPtr joy);
```

Comment:
```suggestion
  void JoyCB(const JoyMsg::SharedPtr joy);
```
you can add `using JoyMsg = sensor_msgs::msg::Bool;` as it is done for other message types